localstack-core/localstack/services/cloudformation/provider_utils.py

```diff
@@ -85,6 +85,27 @@ def _keys_pascalcase_to_lower_camelcase(obj):
     return _recurse_properties(model, _keys_pascalcase_to_lower_camelcase)
 
 
+def keys_pascalcase_to_lower_camelcase_skip_keys(model: dict, exclude_keys: set = None) -> dict:
```

Comment:
We already have `keys_pascalcase_to_lower_camelcase` which is a simplified implementation, so presumably

```python
def keys_pascalcase_to_lower_camelcase(model: dict) -> dict:
    return keys_pascalcase_to_lower_camelcase_skip_keys(model)
```

It might be nice to make this change

Reply:
Maybe something like this?

```python
def test_lower_camelcase_to_pascalcase_skip_keys(self):
    original_dict = {
        "EventBusName": "my-event-bus",
        "Targets": [
            {
                "Id": "AnId",
            }
        ],
    }

    converted_dict = utils.keys_pascalcase_to_lower_camelcase_skip_keys(original_dict, {"id"})
    assert converted_dict == {
        "eventBusName": "my-event-bus",
        "targets": [
            {
                "id": "AnId",
            }
        ],
    }
```
?